.travis.yml

```diff
@@ -1,10 +1,12 @@
-sudo: True
+version: ~> 1.0
 dist: xenial
+os: linux
 language: python
 cache: pip
 python:
-- 3.7
 - 3.6
+- 3.7
+- 3.8-dev
```

Comment:
this should 3.8 only

Reply:
Done. I was not aware that TravisCI added it already.